CHANGES.md

```diff
@@ -1,3 +1,9 @@
+2.6.1 (2022-XX-XX)
+------------------
```

Comment:
Small thing: we're adding a new feature so we should increase the minor version i.e.  `2.6.x` -> `2.7.0`. Note that I usually just write "Next release" instead of trying to guess the next version.